diff --git a/third_party/github.com/bazelbuild/bazel-skylib/rules/private/copy_file_private.bzl b/third_party/github.com/bazelbuild/bazel-skylib/rules/private/copy_file_private.bzl
index 307242d8..d83c350e 100644
--- a/third_party/github.com/bazelbuild/bazel-skylib/rules/private/copy_file_private.bzl
+++ b/third_party/github.com/bazelbuild/bazel-skylib/rules/private/copy_file_private.bzl
@@ -30,7 +30,12 @@ _execution_requirements = {
     # Also, remote-execution does not allow source directory inputs, see
     # https://github.com/bazelbuild/bazel/commit/c64421bc35214f0414e4f4226cc953e8c55fa0d2
     # So we must not attempt to execute remotely in that case.
+    "no-remote": "1",
+    "no-remote-cache": "1",
     "no-remote-exec": "1",
+    "no-cache": "1",
+    "no-sandbox": "1",
+    "local": "1",
 }

 def _hash_file(file):
